File: 백준/Bronze/2480. 주사위 세개/주사위 세개.py
A, B, C = map(int, input().split())

# 조건 1 같은 눈이 3개가 나올 경우
if A == B ==C :
    Money = 10000+A*1000
# 조건 2 같은 눈이 2개만 나오는 경우
elif A == B or A == C :
    Money = 1000 + A*100
elif B == C :
    Money = 1000 + B*100
# 조건 3 모든 다른 눈이 나오는 경우
else :
    dice = [ A, B, C]
    Money = max(dice)*100
    
print(Money)

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.js
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
const input = fs
  .readFileSync(filePath)
  .toString()
  .replaceAll("\r", "")
  .trim()
  .split("\n");


let [a, b, c] = input[0].split(' ').map(Number)

let ans = 0;

const func = ((a, b) => {
  if (a === b)
    return true;
  else
    return false;
})

if (a === b && a === c && c === b) {
  ans = 10000 + a * 1000;
}

else if (a === b || a === c || b === c) {
  if (func(a, b)) ans = 1000 + a * 100;
  else if (func(b, c)) ans = 1000 + b * 100;
  else ans = 1000 + a * 100;
}

else {
  let max = Math.max(a, b, c);
  ans = max * 100;
}

console.log(ans);